zulip/integrations/bridge_with_matrix/matrix_bridge.py

```diff
@@ -75,10 +75,7 @@ def _matrix_to_zulip(room: Any, event: Dict[str, Any]) -> None:
         zulip_bot_user = ('@%s:matrix.org' % matrix_config['username'])
         # We do this to identify the messages generated from Zulip -> Matrix
         # and we make sure we don't forward it again to the Zulip stream.
-        not_from_zulip_bot = (
-            'body' not in event['content']
```

Comment:
Was this a previous way to identify the user as a bot? Is this only in a certain version of the matrix library? If so, should we set a bound on it in `setup.py` with this change?

Reply:
The Matrix Python library hasn't been maintained since 2019, and so has been unchanged. It is the server that is changing its behavior, just like when `membership` key is gone.

Reply:
By the `membership` key bug, I was referring to #612.